CHANGES.md

```diff
@@ -26,9 +26,13 @@ To be released.
 ### Bug fixes
 
  -  Fixed a bug that `Swarm` reported `TaskCanceledException` as an unknown
-   exception while stopping. [[#275]]
+    exception while stopping. [[#275]]
  -  Fixed a bug that `Swarm` didn't stop properly during `Swarm.Preload()`.
-   [[#275]]
+    [[#275]]
+ -  Fixed a bug where the first `TxNonce` of an address is not stripped when
+    forking using `FileStore.ForkTxNonce()` method.  [[#281]]
+ -  Fixed a bug where `LiteDBStore.GetTxNonce()` method throws a
+    `System.IO.IOException` when reading a zero-length file.  [[#281]]
```

Comment:
The expression <q>a zero-length file</q> here seems too much about implementation details.  Actually even I didn't get it (I believe it's relevant to a question I left below, in *LiteDBStore.cs* file).